partiql-tests-data/eval/primitives/operators/bag-operators.ion

```diff
@@ -126,7 +126,6 @@ bagOperators::[
       evalMode:[EvalModeCoerce, EvalModeError],
       result:EvaluationSuccess,
       output:$bag::[
-        1,
```

Comment:
Would be helpful to link https://github.com/partiql/partiql-lang/issues/75 and/or leave a comment about the behavior here. Didn't realize the `DISTINCT` is called on both inputs before calling the `EXCEPT`.